### DISCUSSION TITLE
custom plugin and components for studio

### DISCUSSION THREAD

**Discussion by mredodos** [GrapesJS/studio#10] [General]
Hi i dont understand if is possible to build custom plugin and components for the studio, if yes, how? i don't found the doc.

If no, we need to request here new features?

**Comment by artf** [comment]
Hi @mredodos yeah, we're missing the dedicated page explaining the plugins but for what it's worth it's the same way you would do it with the GrapesJS, eg.
```jsx
<StudioEditor
  options={{
      // ...
      plugins: [
        (editor) => {
          // Use GrapesJS API
          // editor.Blocks.add(.... 
        }
      ]
   }}
/>
```
